### PR TITLE
Remote method to get versions

### PR DESCRIPTION
**Pull request by deepakrkris**
Get loopback versions using a remote method

connect to #448

### CODE REVIEW

File: test/workspace.js [hunk 1]
```diff
@@ -6,11 +6,24 @@
...
 var ConfigFile = app.models.ConfigFile;
 
 describe('Workspace', function() {
+  describe('Workspace.availableLBVersions()', function() {
+    it('Get an array of supported loopback versions', function(done) {
+      request(app).get('/api/workspaces/loopback-versions').expect(200, function(err, res) {
+        if (err) return done(err);
+        expect(res.body).to.not.to.be.undefined();
+        var versions = res.body;
+        expect(versions).to.be.instanceOf(Array);
```
Comment (superkhau) [inline test/workspace.js]:
```
expect(res.body).to.eql({...});
```

instead of the all the checks above

**Comment by anthonyettinger** [comment]
lgtm

File: test/workspace.js [hunk 2]
```diff
@@ -6,11 +6,28 @@
...
+    it('Get an array of supported loopback versions', function(done) {
+      request(app).get('/api/workspaces/loopback-versions').expect(200, function(err, res) {
+        if (err) return done(err);
+        expect(res.body).to.not.to.be.undefined();
+        var versions = res.body.versions;
+        var expectedVersions = [
+          { value: '2.x', description: 'long term support' },
+          { value: '3.x', description: 'current' },
+        ];
+        expect(versions).to.eql(expectedVersions);
```
Comment (superkhau) [inline test/workspace.js]: expect(versions.length).to.equal(2);

File: test/workspace.js [hunk 3]
```diff
@@ -6,11 +6,28 @@
...
+  describe('Workspace.availableLBVersions()', function() {
+    it('Get an array of supported loopback versions', function(done) {
+      request(app).get('/api/workspaces/loopback-versions').expect(200, function(err, res) {
+        if (err) return done(err);
+        expect(res.body).to.not.to.be.undefined();
+        var versions = res.body.versions;
+        var expectedVersions = [
+          { value: '2.x', description: 'long term support' },
+          { value: '3.x', description: 'current' },
+        ];
```
Comment (superkhau) [inline test/workspace.js]: remove all checks above

**Comment by superkhau** [comment]
Reverted this in #452, see follow-up issue at https://github.com/strongloop/loopback-workspace/issues/453

**Pull request merged**